src/static_canvas.class.js

```diff
@@ -755,6 +755,7 @@
       this.calcViewportBoundaries();
       this.clearContext(ctx);
       fabric.util.setImageSmoothing(ctx, this.imageSmoothingEnabled);
+      ctx.patternQuality = 'best';
```

Comment:
Maybe fast as default?

Reply:
fast isn't good enough
look at the diff between `best` and `fast`
![image](https://user-images.githubusercontent.com/34343793/176379504-da40d7ba-39d6-4303-aaad-22a45b565931.png)

Reply:
![objectsInActiveSelections](https://user-images.githubusercontent.com/34343793/176379722-672b807d-60cd-45f1-9f0b-b2fc86504d4c.png)


![objectsInActiveSelections](https://user-images.githubusercontent.com/34343793/176379708-5f4b781e-cf00-42fb-aabc-394e884a00f9.png)